src/OceanSeaIceModels/InterfaceComputations/compute_interface_state.jl

```diff
@@ -17,11 +18,31 @@ end
     return !(converged | reached_maxiter) | hasnt_started
 end
 
+# Fixed number of iterations
 struct FixedIterations
     iterations :: Int
 end
 
-@inline iterating(Ψⁿ, Ψ⁻, iteration, fixed::FixedIterations) = iteration < fixed.iterations
+@inline iterating(Ψⁿ, Ψ⁻, ℙᵢ, iteration, fixed::FixedIterations) = iteration < fixed.iterations
+
+# Stop if the surface temperature is above freezing
+struct SeaIceConvergenceStopCriteria{FT}
+    tolerance :: FT
+    maxiter :: Int
+end
+
+@inline function iterating(Ψⁿ, Ψ⁻, ℙᵢ, iteration, convergence::SeaIceConvergenceStopCriteria)
```

Comment:
I think this is a bad design. We should not have a "sea ice specific convergence criteria". It's not really logical. Keep convergence criteria as actual convergence criteria and figure out another way to express the behavior you want to implement.

Reply:
What do you propose?